examples/prometheus/prometheus.yaml

```diff
@@ -133,24 +142,92 @@ objects:
           imagePullPolicy: IfNotPresent
           volumeMounts:
           - mountPath: /etc/prometheus
-            name: config-volume
+            name: prometheus-config
           - mountPath: /prometheus
-            name: data-volume
+            name: prometheus-data
+
+        # Deploy alertmanager behind prometheus-alert-buffer behind an oauth proxy
```

Comment:
@smarterclayton This works, but I am not sure deploying behind 2 different proxies is the best practice. Is there a way for the oauth-proxy to support 2 incoming ports?